oap-server/analyzer/meter-analyzer/src/main/java/org/apache/skywalking/oap/meter/analyzer/prometheus/rule/Rules.java

```diff
@@ -55,24 +57,24 @@ public static List<Rule> loadRules(final String path, List<String> enabledRules)
         } catch (FileNotFoundException e) {
             throw new ModuleStartException("Load fetcher rules failed", e);
         }
-        final List<String> formedEnabledRules =
-                enabledRules
-                        .stream()
-                        .map(rule -> {
-                            if (rule.startsWith("/")) {
-                                rule = rule.substring(1);
-                            }
-                            if (rule.endsWith(".yaml")) {
-                                return rule;
-                            } else if (rule.endsWith(".yml")) {
-                                return rule.substring(0, rule.length() - 2) + "aml";
-                            } else {
-                                return rule + ".yaml";
-                            }
-                        })
-                        .collect(Collectors.toList());
+        Map<String, Boolean> formedEnabledRules = enabledRules
```

Comment:
Map<String, Boolean> means a Set<String>?

Reply:
What does this value true/false mean?

Reply:
This is used to determine whether an enabled rule does not have a corresponding configuration file

Reply:
At first they were false.
If a rule found its configuration file, it change to true. 
So that I can judge which enabled rule's file is not exist by map's false value